packages/hash/frontend/src/components/BlockContextMenu/BlockLoaderInput.tsx

```diff
@@ -16,7 +12,7 @@ export const BlockLoaderInput: React.VFC = () => {
   const [blockUrl, setBlockUrl] = useState("");
   const blockUrlRef = useRef<HTMLInputElement | null>(null);
 
-  const isDefinedBlock = blocksMeta.has(blockUrl);
+  const isDefinedBlock = blockUrl in blocksMeta;
```

Comment:
This might be safer – `in` has a lot of baggage. 

```suggestion
  const isDefinedBlock = !!blocksMeta[blockUrl];
```

See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in#using_in_with_deleted_or_undefined_properties

Reply:
If not `in` then I'd prefer `hasOwnProperty`. How about it?